pds_doi_service/core/db/doi_database.py

```diff
@@ -217,14 +197,12 @@ def create_q_string_for_transaction_update_is_latest_field(self, table_name):
         """
 
         # Note that we set column "is_latest" to 0 to signify that all previous
-        # rows are now not the latest. The key in dict_row is 'latest_update',
-        # not 'update_date' (not same as column name).
+        # rows are now not the latest.
         o_query_string = f'UPDATE {table_name} '
```

Comment:
My "SQL injection" Spidey-Sense is going off 😅 but this is probably minor, right? `table_name` is a strictly controlled paramater.